test/cpu_test.exs

```diff
@@ -2,15 +2,13 @@ defmodule CPUTaskTest do
   use ExUnit.Case
   alias Beamchmark.Suite.CPU.CPUTask
 
-  setup do
-    # Some tests setup
-    :cpu_sup.start()
-    :cpu_sup.util([:per_cpu])
-    :ok
-  end
-
   test "cpu_task" do
-    cpu_task = CPUTask.start_link(1000, 10)
+    cpu_task =
+      CPUTask.start_link(
+        interval: 10,
+        duration: 1000
+      )
+
     assert {:ok, _statistics} = Task.await(cpu_task, :infinity)
```

Comment:
Could we assert that specific fields in `statistics` are not `nil` and length of cpu_snapshots is correct? Also let's rename `statistics` to `cpu_info`